cluster/charts/rook/templates/clusterrolebinding.yaml

```diff
@@ -14,4 +14,7 @@ subjects:
 - kind: ServiceAccount
   name: rook-operator
   namespace: {{ .Release.Namespace }}
+- kind: ServiceAccount
+  name: rook-agent
```

Comment:
Could you use a different ClusterRoleBinding for the rook-agent.
The rook-agent should not end up with the same permissions as the Rook operator as it is now with this ClusterRoleBinding.

I think it would be good to add a second ClusterRole just for the PSP binding for the `rook-operator`, `rook-agent` and the other rook storage backend component's ServiceAccounts.

Reply:
I did that before & @travisn asked me to combine it :-)

Reply:
hmm, i must have misunderstood the changes, agreed that each component should have minimal privileges. @saravanakumar-periyasamy do you still have the changes to view somewhere? maybe not since it was squashed

Reply:
ya, its squashed now.. I will try to revert it today.